lib/src/form_builder_validators.dart

```diff
@@ -201,25 +201,28 @@ class FormBuilderValidators {
     };
   }
 
+  // TODO(any): l10n
   /// [FormFieldValidator] that requires the field's value to be a valid integer.
-  static FormFieldValidator integer({
+  static FormFieldValidator integer(
+    BuildContext context, {
     String errorText = 'Value must be an integer.',
   }) {
     return (valueCandidate) {
-      if (int.tryParse(valueCandidate) == null && valueCandidate.isNotEmpty) {
+      if (int.tryParse(valueCandidate ?? '') == null) {
         return errorText;
       }
       return null;
     };
   }
 
+  // TODO(any): l10n
   /// [FormFieldValidator] that requires the field's value to be a valid double.
-  static FormFieldValidator double_({
+  static FormFieldValidator double_(
+    BuildContext context, {
     String errorText = 'Value must be a double.',
```

Comment:
`double` is more an _engineering_ term than a _user_ term. Consider revising this `errorText` to be more _user-friendly_, like _"Value must be numeric."_

Reply:
and what do you think about `float` ?

Reply:
Honestly, I think _"double precision"_ and _"floating point"_ are engineering terms more than user friendly terms.  (Would your non-engineer parent or child know what a "double" was?  It is a two-base run in baseball.)

I like "numeric."  That should not be too controversial; people should understand that well enough.

Reply:
Yes it's true.